### PR TITLE
[USH-1151] Data view: Improve user experience when loading more content in split view

### PR DESCRIPTION
**Pull request by Ifycode**
#### Fixes in PR:
Let the user know that some loading is happening before more post cards show up, so that it doesn't look like it's not working (at first):
- [x] Show Loading spinner underneath the postcards on the left (as more posts load), after clicking the "Load More" button.
- [x] Additional: Keep clicking the "Load More" button till all the post cards have loaded - "End of Posts" text shows up underneath the postcards on the left to indicate that there are no more posts to load (since all posts have loaded).

**Plus Fix 🎁🥳:** 
This Fix automatically takes care of these 2 flicker issues:
- Pagination element's brief flicker on the screen when data view is in "tiles mode" (at the point when posts are still loading)
- Load more button's brief flicker on the screen when data view is in "posts mode" (at the point when posts are still loading)

### CODE REVIEW

**Comment by Ifycode** [comment]
PR ready for review

File: apps/web-mzima-client/src/app/feed/feed.component.ts [hunk 1]
```diff
@@ -292,19 +294,23 @@ export class FeedComponent extends MainViewComponent implements OnInit {
     //   this.currentPage = 1;
     // }
     this.isLoading = true;
+    this.loadingMorePosts
+      ? (this.paginationElementsAllowed = true)
+      : (this.paginationElementsAllowed = false); // this check prevents the load more button & area from temporarily disappearing (on click)
```
Comment (ushahidlee) [inline apps/web-mzima-client/src/app/feed/feed.component.ts]: you could replace this with:

this.paginationElementsAllowed = this.isLoading

**Review by ushahidlee — APPROVED** [review]
Looks good and fixes the issue!

File: apps/web-mzima-client/src/app/feed/feed.component.ts (hunk 1 above)
Comment (ushahidlee) [reply apps/web-mzima-client/src/app/feed/feed.component.ts]: sorry, I meant:

this.paginationElementsAllowed = this.loadingMorePosts

Comment (Ifycode) [reply apps/web-mzima-client/src/app/feed/feed.component.ts]: Change made. Thank you @ushahidlee 
https://github.com/ushahidi/platform-client-mzima/pull/800#pullrequestreview-1914293590

Merging @Mh-Asmi

**Pull request merged**